atlp-rwanda/ecommerce-app-legends-bn
# Ch GitHub actions setup

## This commit adds a GitHub actions setup for the project.
- this repo contains a .yml file that will be used to setup
- contains secrets which will be used to run the project
- contains a command to run the testing
- contains a command to install all dependencies for the project
- contains a command to build if it exists
## how to check if the pipeline is well set
- open the specific repository
- click on Actions tab and see if triggered event finished to run 
- red cross sign means there is/are issue(s) in your code
- Green tick sign means that your codes doesn't have any issues
